plugins/renepay/uncertainty.c

```diff
@@ -177,3 +178,22 @@ struct chan_extra *uncertainty_find_channel(struct uncertainty *uncertainty,
 {
 	return chan_extra_map_get(uncertainty->chan_extra_map, scid);
 }
+
+enum renepay_errorcode uncertainty_relax(struct uncertainty *uncertainty,
+					 double seconds)
+{
+	assert(seconds >= 0);
+	const double fraction = MIN(seconds / TIMER_FORGET_SEC, 1.0);
```

Comment:
Does the uncertainty always need to be increased by this call, or could it make sense to return early if it's a minimal duration since the last call?

Reply:
Entropy always increases. If the time since the last call is small compared to `TIMER_FORGET_SEC` then changes will be small.